src/config/settings.js

```diff
@@ -180,8 +180,10 @@ export default {
   window: {
     allowClose: true, // Configure if windows can be closed or not
     allowMaximize: true, // Configure if windows can be maximized or not
+    defaultSideBarPanel: 'info', // Configure which sidebar is selected by default
```

Comment:
```suggestion
    defaultSideBarPanel: 'info', // Configure which sidebar is selected by default. Options: info, rights, canvas, annotations
```

It would be nice to list the available panel names here. Another idea would be linking to [CompanionWindowFactory](https://github.com/ProjectMirador/mirador/blob/287e2ce1938cda8430a4b887c191066fbe5f50a1/src/components/CompanionWindowFactory.js#L17-L29)

Useful follow-up action would be renaming AttributionPanel.js so that it matches the `WindowSidebar{__}Panel` naming convention.

Reply:
I'm trying to move us away from the `WindowSidebar{__}Panel` convention, it's way too wordy. I think we're now 1/2 and 1/2.